packages/gatsby/src/utils/page-data.js

```diff
@@ -35,7 +37,57 @@ const write = async ({ publicDir }, page, result) => {
   await fs.outputFile(filePath, bodyStr)
 }
 
+const getChangedPageDataKeys = (store, cacheData) =>
+  new Promise(resolve => {
```

Comment:
It probably doesn't need to be a promise.

Reply:
And if it does it could be an `async` arrow with regular `return` values and be effectively the same.

But I agree, probably not necessary at all (maybe something that changed during development where it was necessary before and not by the end?).

Reply:
Done =)